src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -530,109 +465,104 @@
 	"item.mmorpg.runes.yon.2": "§9YON - 中品符文",
 	"item.mmorpg.runes.yon.3": "§dYON - 上品符文",
 	"item.mmorpg.runes.yon.4": "§6YON - 精品符文",
-
-"_comment": " [CATEGORY]: RUNES - NAMES",
-
-
-"_comment": " [CATEGORY]: WORDS - NAMES",
-
 	"mmorpg.word.activation_time": "激活时间",
 	"mmorpg.word.addchaosstats": "添加混沌属性",
-	"mmorpg.word.addmajorarcana": "添加较高的奥术混沌属性",
+	"mmorpg.word.addmajorarcana": "添加大阿卡那混沌属性",
```

Comment:
The old translator doesn't know the Major Arcana of Tarot, so they messed up all the Major Arcana translation.